lib/pdf/core.rb

```diff
@@ -16,6 +16,7 @@
 require_relative "core/outline_root"
 require_relative "core/outline_item"
 require_relative "core/renderer"
+require_relative "core/text"
```

Comment:
That was to bring in the PDF::Core::Text class for the specs, the other option is to require it from the spec, or spec_helper, but it seemed odd that it wouldn't be required in the first place.